docs/releases/overview.md

```diff
@@ -26,6 +26,12 @@ The Vega core software is public on a business-source licence, so you can both v
 This version was released to the Vega testnet on 14 July, 2022.
 
 #### 0.53.0 (14 July 2022)
+
+**Smart contract upgrade:**
+With the upgrade of the network to version v0.53 will come an upgrade of the smart contracts. The multisig control contract and the collateral bridge will thus increase users' control over the funds they deposit (opt-out) and include performance improvements, such as decreasing gas cost when using the bridge. The Vega asset pool contract will not be upgraded. Once the new contracts are properly set up on Ethereum, the validators will migrate the asset pool to use the new contracts.
+
+Information on how to upgrade can be found in the [smart contracts migration guide](../node-operators/migration-guides/smart-contracts-migration).
```

Comment:
```suggestion
Information on how to upgrade can be found in the [smart contracts migration guide](../node-operators/migration-guides/smart-contracts-migration.md).
```

Does this need a file name extension to open correctly? When I try this link in your branch it gives a 404, adding the `.md` successfully shows me the file

Reply:
It doesn't but let's see what happens if we do it :)